sql/database.go

```diff
@@ -180,11 +180,7 @@ func Open(uri string, opts ...Opt) (*Database, error) {
 		}
 	}
 	for i := 0; i < config.connections; i++ {
-		conn := pool.Get(context.Background())
-		if err := registerFunctions(conn); err != nil {
-			return nil, err
-		}
-		pool.Put(conn)
+		pool.Put(pool.Get(context.Background()))
 	}
```

Comment:
I think this loop only existed for `registerFunctions` and can be removed completely.